linode/manager
# Fixed styles for the Markdown Accordion of the Ticket modal

## Description

I think #7765 caused a minor regression (shown in screenshot). 

<img width="1000" alt="Screen Shot 2021-07-23 at 11 42 06 AM" src="https://user-images.githubusercontent.com/6440455/126807155-eef1c1ec-4868-40e8-a89a-3e98eec06613.png">

## How to test

- Go to `/support/tickets` 
- Click **Open New Ticket**
- Check styles of the `Formatting Tips` accordion
